back/spec/serializers/web_api/v1/idea_serializer_spec.rb

```diff
@@ -35,4 +35,53 @@
       expect(last_name).to eq 'Thomas Anderson'
     end
   end
+
+  context 'when serializing internal comments count of idea' do
```

Comment:
I think these tests are a bit convoluted and could be simplified, but nothing too bad.